sdk/servicebus/Azure.Messaging.ServiceBus/samples/Sample01_HelloWorld.md

```diff
@@ -0,0 +1,97 @@
+# Sending and Receiving Messages
+
+This sample demonstrates how to send and receive messages from a Service Bus queue.
+
+## Sending and receiving a message
+
+```C# Snippet:ServiceBusSendAndReceive
+string connectionString = "<connection_string>";
+string queueName = "<queue_name>";
+// since ServiceBusClient implements IAsyncDisposable we create it with "await using"
+await using var client = new ServiceBusClient(connectionString);
+
+// get the sender
+ServiceBusSender sender = client.GetSender(queueName);
+
+// create a message that we can send
+ServiceBusMessage message = new ServiceBusMessage(Encoding.Default.GetBytes("Hello world!"));
+
+// send the message
+await sender.SendAsync(message);
+
+// get a receiver that we can use to receive the message
+ServiceBusReceiver receiver = client.GetReceiver(queueName);
+
+// the received message is a different type as it contains some service set properties
+ServiceBusReceivedMessage receivedMessage = await receiver.ReceiveAsync();
+
+// get the message body as a string
+string body = Encoding.Default.GetString(receivedMessage.Body.ToArray());
+Console.WriteLine(body);
+```
+
+### Sending and receiving a batch of messages
+
```

Comment:
Might be nice to add a couple lines to every section highlighting main primitives you are trying to show.

Reply:
Updated